source/install/install-docker.rst

```diff
@@ -4,48 +4,40 @@ Deploy Mattermost via Docker
 .. include:: ../_static/badges/allplans-selfhosted.rst
   :start-after: :nosearch:
 
+.. important::
+   This deployment method is not recommended for production environments. If you're looking to deploy a production Mattermost instance, we recommend using managed container services like AWS ECS, Azure Container Apps, or Google Cloud Run. These services provide better scalability, reliability, and maintenance features suited for production workloads.
```

Comment:
I wonder if this would be better to suggest past a specific size. I can't see why a smaller organization shouldn't run Mattermost via Docker.

Reply:
@spirosoik - Can we get a bit more specific about threshold guidance here?